eLife revise (#117)
greenelab/iscb-diversity-manuscript: content/20.results.md

We included individuals who were honored multiple times as separate entries.
For example, Christine Orengo was a keynote speaker at RECOMB 2004 and became an ISCB Fellow in 2016, and thus was counted twice in this list.

We sought to compare this dataset with a background distribution of potential speakers, which we considered to be last authors of bioinformatics and computational biology manuscripts.
We scraped PubMed for manuscripts written in English from 1993--2019 with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+).
We downloaded the metadata of manuscripts published in these journals from PubMed, which provided 173,735 articles for evaluation.
For each article, we extracted its last author's fore name and last name for analysis.

> In a [previous version](https://greenelab.github.io/iscb-diversity-manuscript/v/711edb01f5335a4c1bb8e1e0575618586a7abf9b/) of this work, instead of last authors, we examined corresponding authors of articles in three well-recognized computational biology journals, and the results were consistent with our findings here.

Reviewer: cite versioned preprint here on biorxiv with URL (since their DOI is not versioned)?

We included individuals who were honored multiple times as separate entries.
For example, Christine Orengo was a keynote speaker at RECOMB 2004 and became an ISCB Fellow in 2016, and thus was counted twice in this list.

> We sought to compare this dataset with a background distribution of potential speakers, which we considered to be last authors of bioinformatics and computational biology manuscripts.

Reviewer 2: This might be a place to take advantage of versioning. You could add one sentence after this one.
> In a previous version of this work we examined corresponding authors instead of last authors, and the results were consistent with our findings here.

Then you could link _version of this work_ to the one before these changes.

We scraped PubMed for manuscripts written in English from 1993--2019 with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+).
We downloaded the metadata of manuscripts published in these journals from PubMed, which provided 173,735 articles for evaluation.

> For each article, we extracted its last author's fore name and last name for analysis.

Reviewer 2: What do you think of taking advantage of versioning here too, and including one-sentence + link to the version of the manuscript before these changes noting that we also examined three well-recognized computational biology journals and found consistent results? We might imagine that journals might appear and disappear, so looking at a fixed set of journals does seem complementary. It's not new work that I'd suggest we do, but since we've already done that it could head off a different type of reader concern.

Author: Great idea. I added it [here](https://github.com/greenelab/iscb-diversity-manuscript/pull/117/commits/fde72e19af4decf5c644a98eb7b748dc622f37ba). I think your other versioning comment is in the same paragraph, so perhaps this one additional sentence is sufficient?

Reviewer 2: Yes! I like it and it shows the power of manubot! Thanks 😁

### Similar gender proportion between ISCB's honorees and the field

> We observed a gradual increase of the proportion of predicted female authors, arriving at 28.8% in 2019 (Fig. {@fig:gender_breakdown}, left).

Reviewer 2: There are more predicted female authors in the broader literature than at those three journals - interesting!

Author: Yes.

In recent years, ISCB Fellows and keynote speakers appear to have similar gender proportion compared to the population of authors published in computational biology and bioinformatics journals (Fig. {@fig:gender_breakdown}, right); however, it has not yet reached parity.
Examining each honor category, we observed an increasing trend of honorees who were women, especially in the group of ISCB Fellows (see [notebook](https://greenelab.github.io/iscb-diversity/10.visualize-gender.html#sup_fig_s1)), which markedly increased after 2015.
Through 2019, there were a number of years when meetings or ISCB Fellow classes have a high probability of recognizing only male honorees and none that appeared to have exclusively female honorees.

> A multiple linear regression of the probability of a name being female on year and the two groups of scientists did not reveal a significant association between the groups and gender predicted from fore names (_p_ = 0.558).

Reviewer 2: I am struggling a bit with this. I think it came from a reviewer comment. What if you said:
We sought to examine XYZ, so we...

I'm worried that a reader who is not the reviewer would get here and be confused about where the analysis came from and what it's aiming to do.

Author: Thank you for this comment. I hope to have clarified it [here](https://github.com/greenelab/iscb-diversity-manuscript/pull/117/commits/2b203db2da6b8ef9d5b1c941d2fbe0d1ac2296a5). Please feel free to tweak further if it's not clear enough.

Reviewer 2: I made a couple suggestions & am curious as to your thoughts :)

Author: Much clearer! Thank you!

![Estimated composition of gender prediction over the years of
  all Pubmed computational biology and bioinformatics journal authors (left),

>   and all ISCB Fellows and keynote speakers (right).s

Reviewer 2: ```suggestion
  and all ISCB Fellows and keynote speakers (right).
```

and the absolute difference between observed (triangle) and expected (circle) number of honorees (right).
Positive value of LOE indicates a higher proportion of honorees affiliated with that country compared to authors.
Countries are ordered based on the proportion of authors in the field.
The overrepresentation of honorees affiliated with institutions and companies in the US and Israel contrasts the underrepresentation of honorees affiliated with those in China, France, Italy, India, the Netherlands, Korea, Brazil and Taiwan.

](https://raw.githubusercontent.com/greenelab/iscb-diversity/f7b8a119faf269f072886bd4e50d533a0913252a/figs/enrichment-plot.png){#fig:country-enrichment width="80%"}

### Overrepresentation of white and underrepresentation of Asian honorees among US-affiliated scientists

We predicted the race and ethnicity of US-affiliated authors and honorees using wru, which is based on US census data.
We found that an increasing proportion of last authors of computational biology papers whose last names associated with selecting Asian as a race/ethnicity category in the US census (Fig. {@fig:us_racial_makeup}A left).

> Compared to Pubmed authors, ISCB honorees with US affiliations have a higher proportion of individuals whose last names we associated with selecting white as a race/ethnicity category in the US census (Fig. {@fig:us_racial_makeup}A).

Reviewer 2: ```suggestion
Compared to Pubmed authors, ISCB honorees with US affiliations have a higher proportion of individuals whose last names were associated with selecting white as a race/ethnicity category in the US census (Fig. {@fig:us_racial_makeup}A).
```

Separating honoree results by honor category did not reveal any clear differences among the categories (see [notebook](https://greenelab.github.io/iscb-diversity/13.us-race-analysis.html#hypothesis-testing).

![Estimated composition of census-based race/ethnicity prediction over the years of
  (A) all Pubmed computational biology and bioinformatics journal authors(left) and all ISCB Fellows and keynote speakers (right),
  (B) For each race/ethnicity category, the mean predicted probability of Pubmed articles is shown as teal generalized additive model (GAM) curve, and the mean probability and 95% confidence interval of the ISCB honoree predictions are shown as dark circles and vertical lines.
  The large difference between the means and the lack of overlaping of the confidence intervals in most years indicates an overrepresentation of white and underrepresentation of Asian honorees as compared to authors.
](https://raw.githubusercontent.com/greenelab/iscb-diversity/f7b8a119faf269f072886bd4e50d533a0913252a/figs/us_racial_makeup.png){#fig:us_racial_makeup}

> ### Reception & Feedback

Reviewer 2: I want to circle back to this a bit, but I will do it on a new branch.

Author: Okay!

In recent years, ISCB Fellows and keynote speakers appear to have similar gender proportion compared to the population of authors published in computational biology and bioinformatics journals (Fig. {@fig:gender_breakdown}, right); however, it has not yet reached parity.
Examining each honor category, we observed an increasing trend of honorees who were women, especially in the group of ISCB Fellows (see [notebook](https://greenelab.github.io/iscb-diversity/10.visualize-gender.html#sup_fig_s1)), which markedly increased after 2015.
Through 2019, there were a number of years when meetings or ISCB Fellow classes have a high probability of recognizing only male honorees and none that appeared to have exclusively female honorees.

> We sought to examine whether there is a difference in the probability of a name being female between the two groups of scientists. 

Reviewer 2: ```suggestion
We sought to examine whether or not there was a difference in the proportion of female names between authors and honorees. 
```

> A multiple linear regression of this probability on the groups and year did not reveal a significant difference (_p_ = 0.558).

Reviewer 2: ```suggestion
A multiple linear regression of this proportion for the groups and year did not reveal a significant difference (_p_ = 0.558).
```
